<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/C:/Users/ganwenzhao/Desktop/notepad-master/app.ico</normaloff>:/C:/Users/ganwenzhao/Desktop/notepad-master/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionRevoke"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionReplace"/>
    <addaction name="actiongoto"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionTime_Date"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format(&amp;Fï¼‰</string>
    </property>
    <addaction name="actionAutoLine"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <addaction name="actionStatusBar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save_As(&amp;A)</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print(&amp;P)</string>
   </property>
  </action>
  <action name="actionRevoke">
   <property name="text">
    <string>Undo(&amp;U)</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut(&amp;T)</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
   <property name="iconText">
    <string>Paste(P)</string>
   </property>
   <property name="toolTip">
    <string>Paste(P)</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete(&amp;L)</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find(&amp;F)</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find_Nxet(&amp;N)</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace(&amp;R)</string>
   </property>
  </action>
  <action name="actiongoto">
   <property name="text">
    <string>Goto(&amp;G)</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select_All(&amp;A)</string>
   </property>
  </action>
  <action name="actionTime_Date">
   <property name="text">
    <string>Time/Date(&amp;D)</string>
   </property>
  </action>
  <action name="actionAutoLine">
   <property name="text">
    <string>AutoLine(&amp;W)</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font(&amp;F)</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>StatusBar(&amp;S)</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help(&amp;H)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit(&amp;X)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
